adapters/gst/gst_plugins/python/media_files_src_bin.py

```diff
@@ -111,6 +119,10 @@ def do_set_state(self, state: Gst.State):
         if self.current_state == Gst.State.NULL and state != Gst.State.NULL:
             assert self.location is not None, '"location" property is required'
             assert self.file_type is not None, '"file-type" property is required'
+            if self.loop_file:
+                assert (
+                    self.file_type == FileType.VIDEO
+                ), f'Only "fil-type={FileType.VIDEO.value}" is allowed when "loop-file" is enabled'
```

Comment:
Typo in 'fil-type'

Reply:
@tomskikh don't forget to update the docs: 
Adapters: https://github.com/insight-platform/Savant/blob/develop/docs/adapters.md
Readme: https://github.com/insight-platform/Savant/
Architecture: https://github.com/insight-platform/Savant/blob/develop/docs/architecture.md